components/camel-hyperledger-aries/src/main/java/org/apache/camel/converter/aries/ConnectionRecordConverter.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
There is a duplicated license header here

Reply:
Fixed